src/css/mobile-main-nav.scss

```diff
@@ -7,23 +7,17 @@
   box-shadow: 0px 3px 15px $shadow-purple;
   border-bottom: 2px solid $shadow-purple;
 
-  @media (min-width: 769px) {
-    display: none;
-  }
+  @media (min-width: 769px) { display: none; }
```

Comment:
pq n if?

Reply:
agora utilizando uma computada com this.$q.screen.gt.sm